src/components/bulkImport/SummaryDetails.tsx

```diff
@@ -0,0 +1,132 @@
+    const currentPage = pagination[activeTab].page;
+    const tabPageSize = pagination[activeTab].pageSize;
+    const tabPageCount = Math.ceil(total / tabPageSize)
+    const displayData = students.slice((currentPage - 1) * tabPageSize, currentPage * tabPageSize);
+
+    const tabClick = (tab: string) => {
+        setActiveTab(tab);
+    };
+    // Calculate the slice of data to display for the current page
+    // const displayData = students.slice((page - 1) * pageSize, page * pageSize);
+    return (<>
+        <TabBar>
+            <Tab onClick={() => {
+                tabClick("new")
+            }} selected={activeTab === 'new'}>
+                {processedRecords.newRecords.length}<br/>
+                New Students
+            </Tab>
+            <Tab onClick={() => {
+                tabClick("updates")
+            }} selected={activeTab === 'updates'}>
+                {processedRecords.recordsToUpdate.length}
+                <br/>
+                Potential Duplicates
+            </Tab>
+            <Tab onClick={() => {
+                tabClick("conflicts")
+            }} selected={activeTab === 'conflicts'}>
+                0<br/> Conflicts
+            </Tab>
+            <Tab onClick={() => {
+                tabClick("invalids")
+            }} selected={activeTab === 'invalids'}>
+                {processedRecords.invalidRecords.length}<br/> Invalid Records
+            </Tab>
+        </TabBar>
+        <br/>
+        <DataTable>
+            <DataTableHead>
+                <DataTableRow>
+                    {/*<DataTableColumnHeader />*/}
+                    <DataTableColumnHeader>Excel ID</DataTableColumnHeader>
+                    <DataTableColumnHeader>School</DataTableColumnHeader>
+                    {
+                        processedRecords.mandatoryFields.map(field => (
+                            <DataTableColumnHeader>{field.name}</DataTableColumnHeader>))
+                    }
+                    <DataTableColumnHeader>Details</DataTableColumnHeader>
+                </DataTableRow>
+            </DataTableHead>
+            <DataTableBody>
+                {(displayData.length > 0) && displayData.map(student => (
+                    <DataTableRow key={student.ref}>
+                        <DataTableCell>{student.ref}</DataTableCell>
+                        <DataTableCell>{student.orgUnitName}</DataTableCell>
+                        {
+                            processedRecords.mandatoryFields.map(field => (
+                                <DataTableColumnHeader>{student[field.key]}</DataTableColumnHeader>))
+                        }
+                        <DataTableCell>{""}</DataTableCell>
+                    </DataTableRow>
+                ))}
+                {(displayData?.length === 0) &&
+                    <DataTableCell>{"No Data"}</DataTableCell>
+                }
+            </DataTableBody>
+        </DataTable>
+        <br/>
+        {total > 0 && <Pagination
```

Comment:
We can use the component already created for pagination. 
[Pagination.tsx](https://github.com/Saudigitus/dhis2-emis-enrollment/blob/develop/src/components/table/components/pagination/Pagination.tsx)